builtin/console.mbt

```diff
@@ -50,7 +50,7 @@ pub fn to_string(self : Int64) -> String {
     if num2 != 0L {
       write_digits(num2)
     }
-    buf.write_char(Char::from_int(abs(num % 10L).to_int() + 48))
+    buf.write_char(Char::from_int_unchecked(abs(num % 10L).to_int() + 48))
   }
```

Comment:
is this in perf critical place? if not, we would prefer safe API

Reply:
no, but we cannot use Option::unwrap in `builtin`